pipeline/pipelines/gapic_generation_pipeline.py

```diff
@@ -98,7 +91,7 @@ def _get_gapic_package_tasks(self, **kwargs):
 class _NodeJSGapicTaskFactory(GapicTaskFactoryBase):
 
     def _get_gapic_package_tasks(self, **kwargs):
-        return [gapic_tasks.GapicMergeTask('GapicMerge', inject=kwargs),
+        return [gapic_tasks.GapicCopyTask('GapicCopy', inject=kwargs),
                 gapic_tasks.GapicPackmanTask('GapicPackman', inject=kwargs)]
 
 
```

Comment:
I think the default copy step would be safe. For 'csharp', final_repo_dir is always set to .../artman/ouput/... . I'll update it.